Description: Use the zlib distributed by Debian, not the one in third-party/zlib-1.2.3/.
Author: Charles Plessy <plessy@debian.org>
Forwarded: no
--- velvet_1.0.02~nozlibcopy.orig/src/readSet.c
+++ velvet_1.0.02~nozlibcopy/src/readSet.c
@@ -32,7 +32,7 @@
 #if defined(_WIN32) || defined(__WIN32__) || defined(WIN32)
 #include "../third-party/zlib-1.2.3/Win32/include/zlib.h"
 #else
-#include "../third-party/zlib-1.2.3/zlib.h"
+#include <zlib.h>
 #endif
 
 #if defined(MSDOS) || defined(OS2) || defined(WIN32) || defined(__CYGWIN__)
